lib/spack/docs/tutorial/examples/1.package.py

```diff
@@ -22,13 +22,12 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
-# flake8: noqa
```

Comment:
Fixed. I opted to use the following syntax to skip the licensing:
```rst
.. literalinclude:: tutorial/examples/1.package.py
   :lines: 25-
   :language: python
```
This skips the beginning of the package and only includes lines 25 to the end.